packages/terra-form/src/Textarea.jsx

```diff
@@ -33,26 +61,44 @@ const propTypes = {
 const defaultProps = {
   defaultValue: undefined,
   name: null,
+  isAutoResizable: false,
+  isInvalid: false,
   onChange: undefined,
   required: false,
+  size: null,
   value: undefined,
 };
 
 const Textarea = ({
-  defaultValue,
   name,
-  onChange,
   required,
+  onChange,
+  onFocus,
+  isAutoResizable,
+  isInvalid,
   value,
+  defaultValue,
+  rows,
+  size,
   ...customProps
 }) => {
   const additionalTextareaProps = Object.assign({}, customProps);
-  const textareaClasses = cx(['textarea', additionalTextareaProps.className]);
+
+  const textareaClasses = cx([
+    'textarea',
+    { 'form-error': isInvalid },
+    { 'full-size': size === 'full' },
+    additionalTextareaProps.className,
+  ]);
 
   if (required) {
     additionalTextareaProps['aria-required'] = 'true';
   }
 
+  const textareaRows = (rows !== undefined) ? rows : TEXTAREA_ROW_SIZES[size];
```

Comment:
It looks like you honor `rows` over textarea `size`. If this is intended, we should probably specify that in the prop description and then give size a default value.

Reply:
[Here](https://github.com/cerner/terra-core/pull/966/files#diff-b5234dd3bb79aebce37d75c701c1a742R6) you say

> /* Smallest Textarea should have at least 2 rows. */

so default to `small`?

Reply:
Fixed it here.

https://github.com/cerner/terra-core/pull/966/commits/87f3e5aa297aa27d054098780307e6cf7ba0cccb